runtime/src/iree/base/internal/file_io.c

```diff
@@ -26,8 +26,17 @@
 
 #define IREE_SET_BINARY_MODE(handle) ((void)0)
 
-#define iree_fseek64 fseek
-#define iree_ftell64 ftell
+#if !defined(fseeko)
+#define iree_fseek64 fseeko64
+#else
+#define iree_fseek64 fseeko
+#endif  // !fseeko
+
+#if !defined(ftello)
+#define iree_ftell64 ftello64
```

Comment:
https://github.com/openxla/iree/actions/runs/5765411327/job/15631311651?pr=14577#step:4:453

```
/work/runtime/src/iree/base/internal/file_io.c:72:21: error: implicit declaration of function 'ftello64' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  uint64_t origin = iree_ftell64(file);
                    ^
/work/runtime/src/iree/base/internal/file_io.c:36:22: note: expanded from macro 'iree_ftell64'
#define iree_ftell64 ftello64
                     ^
```

Reply:
ugh, this is why someone who cares about android should probably do this stuff ;P I'll fiddle until things pass.